HMCLCore/src/main/java/org/jackhuang/hmcl/game/CrashReportAnalyzer.java

```diff
@@ -95,8 +95,11 @@ public enum Rule {
         BLOCK(Pattern.compile("Block: (?<type>.*)[\\w\\W\\n\\r]*?Block location: (?<location>.*)"), "type", "location"),
         // Cannot find native libraries
         UNSATISFIED_LINK_ERROR(Pattern.compile("java.lang.UnsatisfiedLinkError: Failed to locate library: (?<name>.*)"), "name"),
-
-
+        //https://github.com/huanghongxun/HMCL/pull/1813
+        //OPTIFINE_IS_NOT_COMPATIBLE_WITH_FORGE(Pattern.compile("(Cannot read field ""ofTelemetry"" because ""net.optifine.Config.gameSettings"" is null|TRANSFORMER/net.optifine/net\\.optifine\\.reflect\\.Reflector.<clinit>(Reflector.java )")),//OptiFine与Forge不兼容
```

Comment:
这里我注释掉的原因是，我好像不知道如何处理这里的问题，请问该如何除了？

Reply:
`OPTIFINE_IS_NOT_COMPATIBLE_WITH_FORGE(Pattern.compile("(Cannot read field \"ofTelemetry\" because \"net\\.optifine\\.Config\\.gameSettings\\" is null|TRANSFORMER/net\\.optifine/net\\.optifine\\.reflect\\.Reflector\\.<clinit>(Reflector\\.java )"))`

Reply:
@huanghongxun 似乎不对

Reply:
还是不行（